test/integration/test/philips_light_test.js

```diff
@@ -38,57 +35,117 @@ Prepper.makeSuite('Control lights locally',function(){
   });
 
   describe ('once logged in', function () {
+    var lightKeys;
+
     before(function() {
+      // turn off all lights in the simulator
+      Prepper.philipshue_server.turnOffLight(1);
+      Prepper.philipshue_server.turnOffLight(2);
+      Prepper.philipshue_server.turnOffLight(3);
+
       return chakram.post(loginURL,null,{'headers' : header})
       .then(function(resp){
        header.Authorization = 'Bearer ' + resp.body.session_token;
        chakram.setRequestDefaults({'headers': header});
+
+       expect(Prepper.philipshue_server.lightStatus(1) &&
+        Prepper.philipshue_server.lightStatus(2) &&
+        Prepper.philipshue_server.lightStatus(3)).equals(false);
      });
     });
 
     it('check 3 bulbs are registered',function(){
 
-      return chakram.get(serviceListURL)
+      // collect all getters for the lightbulbs
+      var getURL = channelURL + '/get';
+      var getterPayload = [{'kind': 'LightOn'}];
+      
+      return chakram.put(getURL,getterPayload)
       .then(function(listResponse) {
-        expect(listResponse.body.length).equals(3);
+        lightKeys = Object.keys(listResponse.body);
+        expect(lightKeys.length).equals(3);
         expect(listResponse).to.have.status(200);
+
+        //check all lights are reported to be off
+        for (var i =0; i < lightKeys.length; i++){
+          expect(listResponse.body[lightKeys[i]].OnOff).equals('Off');  
+        }
       });
     });
 
-    describe('once the lights are known', function(){
-      before(function() {
-        // turn off all lights in the simulator
-        Prepper.philipshue_server.turnOffLight(1);
-        Prepper.philipshue_server.turnOffLight(2);
-        Prepper.philipshue_server.turnOffLight(3);
-
-        return chakram.get(serviceListURL)
-        .then(function(listResponse) {
-          lightinfo = listResponse.body;
-        });
-      });
+    // Currently, there is no mapping between the foxbox
+    // id and the philips hue id until the tag feature is implemented
+    it('Turn on all lights one by one', function(){
+      var setURL = channelURL + '/set';
+      var lightID = lightKeys[0].replace('getter','setter');
+      var payload = {'select': {'id': lightID}, 'value': { 'OnOff': 'On' } };
+      
+      return chakram.put(setURL,payload)
```

Comment:
Just like for [the getters](https://github.com/fxbox/foxbox/pull/382/files#r59843302), I think this call is at the same level of abstraction than the other functions in `philipshue_server.js`, I'd put it there, I think.